src/components/composer/composer/composer.ts

```diff
@@ -48,7 +48,7 @@ css/* scss */ `
     padding: 0;
     margin: 0;
     border: 0;
-    z-index: ${ComponentsImportance.Composer};
+    z-index: ${ComponentsImportance.Composer + 1};
```

Comment:
> seems useless. probably is. If the composer cannot use its own composer importance, it would mean that the value set in the enum in the first place was wrong.

The issue here is that the formula-helper-assistant is being overlayed by grid composer even though using the same component importance enum so giving the +1 to the composer in topbar, observing the Gsheet's behaviour assistant should be over composer. check the screenshot below 
![image](https://user-images.githubusercontent.com/121081869/224960277-2224d3db-fe65-40a8-8c00-5fcae19e6913.png) issue is only in 16.0 , 16.1 not in master. please look into it.

Reply:
my comment still stands, if the ComponentsImportance.composer needs to be incremented, then it has the wrong value. why is this issue present to start with ? because the component importance is messed up to start with. you have both composers on the same z-index, which means taht their visibility is defined by their position in the dom (here the grid composer is added after the top bar composer so it is above the top bar composer.

For the record, you are not incrementing the zindex of the topbar, you are incrementing the zindex of any composer, it just so happens that the DOM structure is such that the comoser of the grid inherits of the z-index of its parent 'GridComposer' which is set to ` ComponentsImportance.composer` which takes precedence on the z-index defined on the component `Composer`, hence why the fix you suggest works.

This does not mean that the fix is right. If you look at what I just described, you should conlude that the way we defined the componentImportance is crooked because when we have both  composers visible at the same time, one should take precedence on the other, => that would suggest the need of two distinct entries in `ComponentsImportance`.